cryptography/src/lib.rs

```diff
@@ -206,6 +206,16 @@ pub trait Hasher: Clone + Send + Sync + 'static {
 
     /// Return result of hashing nothing.
     fn empty() -> Self::Digest;
+
+    /// A convenience method for hashing a single message.
+    ///
+    /// This should have the same behavior as:
+    /// ```ignore
```

Comment:
nit: if we add a code snippet to the docs, we require it to be runnable to ensure it doesn't drift.